### PR TITLE
Article: She Code Africa Contributhon 2022

### PR DESCRIPTION
**Pull request by mzoyinda**
**Description**

This PR fixes #[SCA 2022 Blog Post] #2786
blog2
blog1

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
This is good to see. :)

**Review by Ruth-ikegah — APPROVED** [review]
LGTM!

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js [hunk 1]
```diff
@@ -103,6 +103,12 @@ const ProjectItemWrapper = styled.section`
           h4 {
             font-size: 0.7rem;
           }
+      }
+      @media screen and (max-width: 660px){
```
Comment (kayceeDev) [inline src/sections/Home/Proud-maintainers/proudMaintainers.style.js:107]: What does this CSS do?

**Review by kayceeDev — CHANGES_REQUESTED** [review]
Looking at the preview? The blog content is padded and doesn't align well with its title. Take reference from other blog posts.

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js (hunk 1 above)
Comment (mzoyinda) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:107]: I didn't add any styling to the blog, any styling you see there was auto-generated from the blog wrapper

Comment (mzoyinda) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:107]: besides, there is no h4 tag in the index.mdx

**Comment by mzoyinda** [comment]
> Looking at the preview? The blog content is padded and doesn't align well with its title. Take reference from other blog posts.

I guess its because the blog's title is long, I didn't add any styling
check out this blog with a long title, 
 https://62961a9867999b5843513443--layer5ng.netlify.app/blog/programs/google-summer-of-code-2020-service-mesh-performance-with-envoy-nighthawk

**Comment by kayceeDev** [comment]
> > Looking at the preview? The blog content is padded and doesn't align well with its title. Take reference from other blog posts.
> 
> I guess its because the blog's title is long, I didn't add any styling check out this blog with a long title, https://62961a9867999b5843513443--layer5ng.netlify.app/blog/programs/google-summer-of-code-2020-service-mesh-performance-with-envoy-nighthawk

It has a long title but it is aligned with the blog contents.

**Review by kayceeDev — APPROVED** [review]

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js (hunk 1 above)
Comment (Chadha93) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:107]: @mzoyinda Colud you please revert only your last commit, it's taking from others commit. We are good to merge then.

Comment (Nikhil-Ladha) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:107]: These changes are not required, I assume.
Can you please revert it @mzoyinda .

File: src/sections/Meshmap/meshmap-modes.js [hunk 2]
```diff
@@ -255,7 +255,7 @@ const MeshmapModes = () => {
             <p>Create and share your own Kubernetes deployments and <Link to="/learn/service-mesh-books/service-mesh-patterns">service mesh patterns</Link> in MeshMap Designer or import from the catalog. </p>
           </div>
         </div>
-      </section>
+      </section> */}
```
Comment (adithyaakrishna) [inline src/sections/Meshmap/meshmap-modes.js:258]: Can we remove this section, if it's not needed as it is commented?

**Comment by l5io** [comment]
🚀 Preview for commit b094de0b762986687e4886a8d203a028a54e5127 at: https://629998435e84691eccf011e4--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looking good @mzoyinda 👌🏼

**Pull request merged**